datasets/dataset_loading_hub.py

```diff
@@ -44,12 +55,66 @@ def get_data(self, data_filepath, val_set_percentage, download=False):
         )
 
         test_set = datasets.MNIST(
-            data_filepath, train=False, transform=self.transform_validate
+            data_filepath,
+            train=False,
+            transform=transform_validate,
         )
         num_labels = 10
         return train_set, val_set, test_set, num_labels
 
 
+class EMNISTLoader:
+    def get_default_transforms(self):
+        normalize = transforms.Normalize(mean=[0.1307], std=[0.3081])
+
+        transform_train = [
+            transforms.ToTensor(),
+            normalize,
+        ]
+
+        transform_validate = [
+            transforms.ToTensor(),
+            normalize,
+        ]
+
+        return transform_train, transform_validate
+
+    def get_data(
+        self, data_filepath, val_set_percentage, download=False
+    ):
+        self.image_shape = ImageShape(
```

Comment:
I guess we should upgrade `ImageShape` from named tuple to a data class.

Reply:
Alrighty. On it.